psi4/driver/driver_cbs_helper.py

```diff
@@ -560,39 +626,30 @@ def corl_xtpl_helgaker_2(functionname: str, zLO: int, valueLO: float, zHI: int,
         return final
 
     elif isinstance(valueLO, (core.Matrix, core.Vector)):
+        valueLO = np.array(valueLO)
+        valueHI = np.array(valueHI)
 
-        beta = valueHI.clone()
-        beta.subtract(valueLO)
-        beta.scale(1 / (zHI**(-alpha) - zLO**(-alpha)))
-        beta.name = 'Helgaker Corl (%s, %s) beta' % (zLO, zHI)
-
-        value = valueHI.clone()
-        value.scale(zHI**alpha)
-
-        tmp = valueLO.clone()
-        tmp.scale(zLO**alpha)
-        value.subtract(tmp)
-
-        value.scale(1 / (zHI**alpha - zLO**alpha))
-        value.name = 'Helgaker Corr (%s, %s) data' % (zLO, zHI)
+        value = (valueHI * zHI**alpha - valueLO * zLO**alpha) / (zHI**alpha - zLO**alpha)
+        beta = (valueHI - valueLO) / (zHI**(-alpha) - zLO**(-alpha))  # lgtm[py/unused-local-variable]
```

Comment:
If this variable is unused, why keep it? Same comment applies for every `lgtm[py/unused-local-variable]`

Reply:
nice for printing, if you want it. I can print, so it's not unused.

Reply:
Please do.